src/main/java/Chat/tasks/Evelyn.java

```diff
@@ -0,0 +1,261 @@
+                tasks.add(new Deadlines(description, by));
+                indexOfTask++;
+                printAddingWords();
+                printLine();
+            } catch (ArrayIndexOutOfBoundsException e){
+                System.out.println("Please enter a description for the deadline!");
+            }
+        } else {
+            System.out.println("Please enter a time in the format of 'by/ (time)'");
+        }
+    }
+
+    private static void todo(String line) {
+        try {
+            tasks.add(new Todos(line.substring(5).trim()));
+            indexOfTask++;
+            printAddingWords();
+            printLine();
+        } catch(StringIndexOutOfBoundsException e){
+            System.out.println("Please enter a description for the todo task!");
+        }
+    }
+
+    private static void unmark(String line) {
+        try {
+            int index = Integer.parseInt(line.substring(7).trim()) - 1;
+            try {
+                markTask(index, false);
+            }
+            catch (RepeatMark | RepeatUnmark e){
+                System.out.println("This task is already marked.");
+            }
+            catch(InvalidIndex e){
+                System.out.println("Invalid task index, please try again!");
+            }
+        } catch(NumberFormatException e){
+            System.out.println("Please key in a number after 'unmark'");
+        }
+    }
+
+    private static void mark(String line) {
+        try {
+            int index = Integer.parseInt(line.substring(5).trim()) - 1;
+            try {
+                markTask(index, true);
+            } catch (InvalidIndex e){
+                System.out.println("Invalid task index, please try again!");
+            } catch (RepeatMark | RepeatUnmark e){
+                System.out.println("This task is already marked.");
+            }
+        } catch (NumberFormatException e){
+            System.out.println("Please key in a number after 'mark'");
+        }
+    }
+
+    private static void printAddingWords() {
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + tasks.get(indexOfTask - 1));
+        System.out.println("Now you have " + indexOfTask + " tasks in the list.");
+    }
+
+    public static void printLine() {
+        System.out.print("____________________________________________________________\n");
+    }
+
+    public static void greeting() {
+        printLine();
+        System.out.println("Hello! I'm Evelyn");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void end() {
+        System.out.println("Bye. Hope to see you again soon!");
+        printLine();
+    }
+
+    public static void readFile(ArrayList<Task> tasks) throws FileNotFoundException {
+        File file = new File("tasks.txt");
+        if(file.exists()){
+            try{
+                Scanner sc = new Scanner(file);
+                while(sc.hasNext()){
```

Comment:
```suggestion
                while (sc.hasNext()) {
```

Take note of the spacing